pool/db_test.go

```
@@ -24,7 +221,7 @@ func testInitDB(t *testing.T) {
 	}
 	defer td()
 
-	// Ensure the db bucket have been created.
+	// Ensure the db buckets have been created.
 	err = db.View(func(tx *bolt.Tx) error {
```

This error is never checked.